### PR TITLE
Build: Add code for strict mode

### PR DESCRIPTION
**Pull request by dmethvin**
Not sure if I should have any other tests here. I don't need or need to put all the tests into strict mode.

### CODE REVIEW

**Comment by dmethvin** [comment]
Duh fixes #179

**Comment by mgol** [comment]
Interesting, no changes required to pass the tests? :)

LGTM then!

File: src/intro.js [hunk 1]
```diff
@@ -1,3 +1,2 @@
 (function( jQuery, window, undefined ) {
```
Comment (mgol) [inline src/intro.js]: BTW, we could drop the `undefined`; Core doesn't protect itself from `undefined` shadowing anymore so there's no reason Migrate should.

**Comment by dmethvin** [comment]
I tried it on all the desktop browsers and Android 4.1, they seemed to be fine. I even tried it on IE8 not really thinking. It doesn't work there. :stuck_out_tongue: We'll see if the whole suite has any issues.

**Pull request closed without merging**